doc/nrf/libraries/zigbee/zigbee_fota.rst

```diff
@@ -95,6 +95,21 @@ The Zigbee FOTA library has the following limitations:
 * The Zigbee FOTA upgrades are currently only supported on the nRF52840 DK (PCA10056) and nRF5340 DK (PCA10095).
 * The Zigbee FOTA library does not currently support bootloader upgrades.
 
+Additionally, the following limitations apply on the nRF5340 MCUs:
+
+* It is required to use external flash to enable Zigbee FOTA library.
+* By default, only the full (application and network core) upgrades are allowed.
+  Disable the :kconfig:option:`CONFIG_NRF53_ENFORCE_IMAGE_VERSION_EQUALITY` Kconfig option to build update images without
+  inter-dependencies so that they can be applied independently.
```

Comment:
```suggestion
  Disable the :kconfig:option:`CONFIG_NRF53_ENFORCE_IMAGE_VERSION_EQUALITY` Kconfig option to build update images without inter-dependencies, so that they can be applied independently.
```

Reply:
fixed